tools/depends/target/libandroidjni/Makefile

```diff
@@ -3,7 +3,7 @@ DEPS= ../../Makefile.include Makefile
 
 # lib name, version
 LIBNAME=libandroidjni
-VERSION=aa12538cc5090d061d34b5fd7385d939ea82ce8b
+VERSION=fc24c9cc507a244a587fa590f6882a588b9fc927
```

Comment:
Does this PR rely need the following changes https://github.com/xbmc/libandroidjni/compare/aa12538...25258a1?

Reply:
This PR needs the changes of fc24c9cc507a244a587fa590f6882a588b9fc927

Reply:
Yes this PR needs the code changes of commit https://github.com/xbmc/libandroidjni/commit/fc24c9c. But does it need the code changes happening in https://github.com/xbmc/libandroidjni/compare/aa12538...25258a1 (memory and hdr info)?

Reply:
No, it doesn't need them, now I'm curious about your suggestion....

Reply:
Then use a version which is based on https://github.com/xbmc/libandroidjni/commit/aa12538 and includes the code changes of https://github.com/xbmc/libandroidjni/commit/fc24c9c.